src/components/ChatLog.jsx

```diff
@@ -0,0 +1,46 @@
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+
+const ChatLog = (props) => {
+  const chatentrycomponents = props.entries.map((entries, index) => {
+    return (
+      <ul key={index}>
```

Comment:
Since we have the `id`s of entries we could actually use that. We want to avoid using the indexes.